packages/suite/src/middlewares/suite/sentryMiddleware.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
+import { MiddlewareAPI } from 'redux';
+import { DEVICE } from '@trezor/connect';
+
+import { SUITE, ROUTER } from '@suite-actions/constants';
+import { getFwVersion, getBootloaderVersion } from '@suite-utils/device';
+import { getSuiteReadyPayload } from '@suite-utils/analytics';
+import { setSentryContext, setSentryTag } from '@suite-utils/sentry';
+
+import type { AppState, Action, Dispatch } from '@suite-types';
+
+const deviceContextName = 'trezor-device';
+
+const sentryMiddleware =
+    (api: MiddlewareAPI<Dispatch, AppState>) => (next: Dispatch) => (action: Action) => {
+        // pass action
+        next(action);
+
+        const state = api.getState();
+
+        switch (action.type) {
+            case SUITE.INIT:
```

Comment:
```suggestion
            case SUITE.READY:
```

To be consistent with analytics. I changed it SuiteReady PR

Reply:
01d123d7787082f566999b2bf2802e582a9af31c